bin/pyAFQ_dki

```diff
@@ -28,7 +28,9 @@ parser.add_option("-o", "--out_dir",
                   help="full path to directory for files to be saved (will be created if it doesn't exist)")
 
 parser.add_option("-m", "--mask", help="mask file")
+parser.add_option('-b', '--b0_threshold', help="b0 threshold", default=0)
+
 opts, args = parser.parse_args()
 
 dki.fit_dki(opts.dwi, opts.bval, opts.bvec, mask=opts.mask,
-            out_dir=opts.out_dir)
+            out_dir=opts.out_dir, b0_threshold=float(opts.b0_threshold))
```

Comment:
Like so?
https://github.com/yeatmanlab/pyAFQ/pull/28/commits/1be852b5a1f27a15544b021c1a17dc7cbb54291b

On Tue, Jun 7, 2016 at 1:36 PM, Ben Cipollini <notifications@github.com>
wrote:

> In bin/pyAFQ_dki
> <https://github.com/yeatmanlab/pyAFQ/pull/28#discussion_r66147712>:
>
> >  opts, args = parser.parse_args()
> >
> >  dki.fit_dki(opts.dwi, opts.bval, opts.bvec, mask=opts.mask,
> > -            out_dir=opts.out_dir)
> > +            out_dir=opts.out_dir, b0_threshold=float(opts.b0_threshold))
>
> I think you'd just change to:
> parser.add_option('-b', '--b0_threshold', help="b0 threshold",
> type="float", default=0)
>
> Then do away with the float conversion later. In theory :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yeatmanlab/pyAFQ/pull/28/files/4be383a0a0b63068500253fb9c52826fece45f26#r66147712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHPNrJEXVj3R68Wp42LnV88TxtzB6vVks5qJdZMgaJpZM4IwQmX>
> .
>